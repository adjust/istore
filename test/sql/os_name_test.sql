BEGIN;
-- os_name should pass valid os_name;
-- ./spec/os_name_test_spec.rb:8;
CREATE EXTENSION istore;
SELECT 'android'::os_name;
SELECT 'ios'::os_name;
SELECT 'windows'::os_name;
SELECT 'windows'::os_name;
SELECT 'unknown'::os_name;
ROLLBACK;
BEGIN;
-- os_name should rais an exception on invalid os_name;
-- ./spec/os_name_test_spec.rb:14;
CREATE EXTENSION istore;
SELECT 'kermit'::os_name;
ROLLBACK;
