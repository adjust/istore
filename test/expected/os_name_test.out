BEGIN;
-- os_name should pass valid os_name;
-- ./spec/os_name_test_spec.rb:8;
CREATE EXTENSION istore;
ERROR:  type "istore" already exists
SELECT 'android'::os_name;
ERROR:  current transaction is aborted, commands ignored until end of transaction block
SELECT 'blackberry'::os_name;
ERROR:  current transaction is aborted, commands ignored until end of transaction block
SELECT 'ios'::os_name;
ERROR:  current transaction is aborted, commands ignored until end of transaction block
SELECT 'linux'::os_name;
ERROR:  current transaction is aborted, commands ignored until end of transaction block
SELECT 'macos'::os_name;
ERROR:  current transaction is aborted, commands ignored until end of transaction block
SELECT 'server'::os_name;
ERROR:  current transaction is aborted, commands ignored until end of transaction block
SELECT 'symbian'::os_name;
ERROR:  current transaction is aborted, commands ignored until end of transaction block
SELECT 'webos'::os_name;
ERROR:  current transaction is aborted, commands ignored until end of transaction block
SELECT 'windows'::os_name;
ERROR:  current transaction is aborted, commands ignored until end of transaction block
SELECT 'windows-phone'::os_name;
ERROR:  current transaction is aborted, commands ignored until end of transaction block
SELECT 'unknown'::os_name;
ERROR:  current transaction is aborted, commands ignored until end of transaction block
ROLLBACK;
BEGIN;
-- os_name should rais an exception on invalid os_name;
-- ./spec/os_name_test_spec.rb:26;
CREATE EXTENSION istore;
ERROR:  type "istore" already exists
SELECT 'kermit'::os_name;
ERROR:  current transaction is aborted, commands ignored until end of transaction block
ROLLBACK;
